examples/MNIST/image_file.sh

```diff
@@ -0,0 +1,7 @@
+cat > image.csv << EOF
+key, image
+four, image1.png
+five, image2.png
```

Comment:
```suggestion
five, image2.mnistpng
```

Reply:
Similarly to how the tensorflow example tests work (See https://github.com/intel/dffml/commit/e7aaeb1e1e2f1b2ce74d1fe6ecc6422bf6df178c#diff-7d57b4e9044e2e8b28e777c8526cdd4e) we should make this example tested too. Copy that file and use it as a template for your tests. Change the function that makes the [csv files](https://github.com/intel/dffml/commit/e7aaeb1e1e2f1b2ce74d1fe6ecc6422bf6df178c#diff-7d57b4e9044e2e8b28e777c8526cdd4eR21-R22) to make sure it copies the `examples/MNIST/image*.png` files into the temporary directory too.

You'll want to find the path to the image relative to the test file.

So if the test file is at

`examples/MNIST/test_mnist.py`

```python
@contextlib.contextmanager
def directory_with_csv_files():
    with tempfile.TemporaryDirectory() as tempdir:
        with chdir(tempdir):
            subprocess.check_output(["bash", sh_filepath("train_data.sh")])
            subprocess.check_output(["bash", sh_filepath("test_data.sh")])
            for image in pathlib.Path(__file__).parent.glob("*.png"):
                shutil.copy(str(image.absolute()), image.name)
            yield tempdir
```